static/js/admin/src/MoveDropLocation.js

```diff
@@ -50,27 +50,40 @@ define('MoveDropLocation', [ 'jquery', 'jquery-ui', 'ConfirmationDialog'],
 				
 				// Confirm the move operation before performing it
 				var representative = ui.draggable.data("resultObject");
-				
-				var repTitle = representative.metadata.title;
-				if (repTitle.length > 50) {
-					repTitle = repTitle.substring(0, 50) + "...";
-				}
-				
-				var destTitle = metadata.title;
-				if (destTitle.length > 50) {
-					destTitle = destTitle.substring(0, 50) + "...";
+				var promptText = '';
+				var moveSingleObj = self.manager.dragTargets.length === 1;
+
+				var destTitle = self._formatTitle(metadata.title);
+				var targetAdminUnit = self._getAdminUnit(metadata.objectPath);
+
+				var repTitle = self._formatTitle(representative.metadata.title);
+				var currentAdminUnit = self._getAdminUnit(representative.metadata.objectPath);
+
+				if (currentAdminUnit.pid !== targetAdminUnit.pid) {
```

Comment:
I think this would throw an error if the objectPath didn't have a length > 1 since it returns undefined. I think this could occur if you were moving a collection to a different unit